package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "todo-list-project",
+  "version": "1.0.0",
+  "description": "this is a simple todo list application using webpack and it is being served by webpack dev server",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack-dev-server --mode production --entry ./src/js/index.js --output-filename ./dist/bundle.js",
+    "build:prod": "webpack src/js/index.js dist/bundle.js -p",
+    "start": "webpack serve"
```

Comment:
- [ ] OPTIONAL
Your local server does open automatically when the command `npm start` is run. Kindly modify the start option to be:
`"start": "webpack serve --open",` 
to enable the page load automatically in the browser.